js/src/modal.js

```diff
@@ -188,7 +188,7 @@ class Modal extends BaseComponent {
     this._element.classList.add(CLASS_NAME_SHOW)
 
     const transitionComplete = () => {
-      if (this._config.focus) {
+      if ("undefined" != typeof this._config.focus && this._config.focus) {
```

Comment:
we don't use yoda comparisons, please change it to normal ones and make sure lint passes